circuits/cpp/barretenberg/ts/src/main.ts

```diff
@@ -140,12 +143,11 @@ export async function contract(outputPath: string, vkPath: string) {
   try {
     await api.acirLoadVerificationKey(acirComposer, new RawBuffer(readFileSync(vkPath)));
     const contract = await api.acirGetSolidityVerifier(acirComposer);
-    if (outputPath === '-') {
-      console.log(contract);
-    } else {
-      writeFileSync(outputPath, contract);
-      console.log(`contract written to: ${outputPath}`);
-    }
```

Comment:
This was a kind of clumsy way of doing this but we should try to support calling commands without writing to file (granted we can just write to `/dev/null` but that's unix specific.)

Reply:
This can be done in a follow up though.

Reply:
Can you open up an issue for it?

Reply:
https://github.com/AztecProtocol/aztec-packages/issues/1693